go.mod

```diff
@@ -193,53 +193,55 @@ require (
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.5
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.5
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.5
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.5
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.5
-	k8s.io/kubectl => k8s.io/kubectl v0.20.5
-	k8s.io/kubelet => k8s.io/kubelet v0.20.5
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.5
-	k8s.io/metrics => k8s.io/metrics v0.20.5
-	k8s.io/mount-utils => k8s.io/mount-utils v0.20.3-rc.0
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.5
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.20.5
-	k8s.io/sample-controller => k8s.io/sample-controller v0.20.5
+	k8s.io/api => k8s.io/api v0.20.11
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.11
+	k8s.io/apimachinery => k8s.io/apimachinery v0.20.11
+	k8s.io/apiserver => k8s.io/apiserver v0.20.11
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.11
+	k8s.io/client-go => k8s.io/client-go v0.20.11
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.20.11
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.11
+	k8s.io/code-generator => k8s.io/code-generator v0.20.11
+	k8s.io/component-base => k8s.io/component-base v0.20.11
+	k8s.io/component-helpers => k8s.io/component-helpers v0.20.11
+	k8s.io/controller-manager => k8s.io/controller-manager v0.20.11
+	k8s.io/cri-api => k8s.io/cri-api v0.20.11
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.11
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.11
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.11
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.11
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.11
+	k8s.io/kubectl => k8s.io/kubectl v0.20.11
+	k8s.io/kubelet => k8s.io/kubelet v0.20.11
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.11
+	k8s.io/metrics => k8s.io/metrics v0.20.11
+	k8s.io/mount-utils => k8s.io/mount-utils v0.20.11
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.11
+	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.20.11
+	k8s.io/sample-controller => k8s.io/sample-controller v0.20.11
 )
 
 replace gopkg.in/DataDog/dd-trace-go.v1 => gopkg.in/DataDog/dd-trace-go.v1 v1.30.0
 
+// Remove once the PR aptly-dev/aptly#967 is merged and released.
+replace github.com/aptly-dev/aptly => github.com/lebauce/aptly v0.7.2-0.20210723103859-345a32860f4d
+
 // Remove once the PR kubernetes/kube-state-metrics#1516 is merged and released.
 replace k8s.io/kube-state-metrics/v2 => github.com/ahmed-mez/kube-state-metrics/v2 v2.1.0-rc.0.0.20210629115837-e46f17606d22
 
-// Remove once the PR aptly-dev/aptly#967 is merged and released.
-replace github.com/aptly-dev/aptly => github.com/lebauce/aptly v0.7.2-0.20210723103859-345a32860f4d
+replace github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt v3.2.1+incompatible
```

Comment:
Can you add a comment explaining why we need this so that we know when we can remove it?